VM/include/lualib.h

```diff
@@ -25,6 +25,9 @@ LUALIB_API const char* luaL_optlstring(lua_State* L, int numArg, const char* def
 LUALIB_API double luaL_checknumber(lua_State* L, int numArg);
 LUALIB_API double luaL_optnumber(lua_State* L, int nArg, double def);
 
+LUALIB_API bool luaL_checkboolean(lua_State* L, int narg);
```

Comment:
This and optboolean should use `int` instead of `bool` to make sure that the header is C-compatible (we'll start validating that in CI in the future)